runtime/src/runtime.c

```diff
@@ -91,6 +95,11 @@ runtime_set_resource_limits_to_max()
 void
 runtime_initialize(void)
 {
+	runtime_worker_threads          = malloc(runtime_worker_threads_count * sizeof(pthread_t));
```

Comment:
Any reason not to use `calloc` here as well? I believe there is an assert like the following elsewhere:

```c
assert(runtime_worker_threads[i] != 0);
```

Reply:
Ahh, I see you removed that below. Was there a specific reason?

Reply:
I removed it because at this point the thread ids cannot be uninitialized, since we check successful `pthread_create` right after each create.

Reply:
Assertions are there to validate and document assumptions. It is rare to make an assumption and use that as a justification to remove an assumption.

Reply:
Agreed. Changing `malloc` to `calloc`.